frontend/src/components/chatInput/useChatInput.ts

```diff
@@ -1,28 +1,32 @@
-import { ChangeEvent, FormEvent, useState } from "react";
+import { useFormik } from "formik";
+import * as Yup from "yup";
 
-const useChatInput = () => {
-    const [currentInputValue, setCurrentInputValue] = useState("");
+export const messageRegExp =
+    /(https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.\S{2,}|www\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.\S{2,}|https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9]+\.\S{2,}|www\.[a-zA-Z0-9]+\.\S{2,})/;
+export const maxCount = 500;
 
-    const handleSetCurrentInputValue = (
-        event: ChangeEvent<HTMLInputElement>
-    ) => {
-        setCurrentInputValue(event.target.value);
-    };
-
-    const handleSubmit = (event: FormEvent<HTMLFormElement>) => {
-        event.preventDefault();
-
-        if (currentInputValue) {
-            // TODO: send request through WS
-            setCurrentInputValue("");
+const useChatInput = (imageUrl = "https://images.com/hbg23vhgvansd") => {
```

Comment:
U have avatar with whole user in global state, if it's null, then u can use this value, but don't create in as keyword args, but as const, cause u didn't want to change default avatar when app working.